docs/test_runners.md

```diff
@@ -0,0 +1,48 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+1. Freshness (Valoraremos esto con `snyk`).
+2. Como de completo es el `framework` para satisfacer las necesidades del proyecto.
+
+## Opciones
+> Tras leer en varios foros y teniendo en cuenta las características del modelo, las siguientes son las opciones que tendremos en consideración pese a que haya muchas alternativas más.
+
+Destacaremos `frameworks` centrados en *BDD* (Behavior Driven Development). Estos resultan más prácticos teniendo en cuneta la complejidad de las clases a testear, ya que facilitan la posibilidad de agrupar comprobaciones en subtests, lo cuál será necesario en nuestro proyecto. Estos frameworks suelen traer sus propias librerías de aserciones. Este tipo de enfoque nos permite describir el comportamiento esperado de una forma bastante entendible.
+
+### Chai.js
```

Comment:
Y esto es una biblioteca de aserciones, ¿no?

Reply:
Tiene "estilos" BDD, lo he leido en su manual. Le adjunto un enlaces a donde se menciona esto:
1. https://www.chaijs.com/
2. https://www.chaijs.com/api/bdd/

Reply:
Si tiene estilos, es una biblioteca de aserciones. Da igual que use `expct`, `it` o `assert`. O que digan que es un *matcher*. Se trata de comparar salida deseada con salida obtenida.